app/views/lks/form.html

```diff
@@ -0,0 +1,36 @@
+{% extends "lks-layout.html" %}
+{% block pageTitle %}
+Home - HEE
+{% endblock %}
+
+{% block content %}
+
+<form class="form" method="post" name="newsletter">
```

Comment:
@thojava - As discussed, could we rewrite the form using `nunjucks` (like the following) for consistency ?

```
<form class="form" method="post" name="newsletter">
    <h2>Subscribe to our newsletter</h2>
    {{ input({
      "label": {
        "text": "Full name"
      },
      "id": "fullName",
      "name": "fullName",
      "autocomplete": "name",
      "errorMessage": {
        "text": "Error message goes here"
      }
    }) }}
    {{ input({
      "label": {
        "text": "Email address"
      },
      "hint": {
        "text": "Your business email address"
      },
      "id": "email",
      "name": "email",
      "autocomplete": "email"
    }) }}
    {{ button({
      "text": "Continue"
    }) }}
</form>
```

The above seem to add visually hidden `Error:` to the field error and `aria-describedby` for accessibility.

Thanks!